python/lsst/pipe/supertask/activator.py

```diff
@@ -313,84 +357,86 @@ def load_super_task(super_taskname):
-                    super_taskname.replace('Task', 'Config')), None)
+                ' No SuperConfig Class %s found: Task or similiar' % (super_taskname.replace('Task',
+                                                                                             'Config')), None)
 
         return class_task_instance
 
-
     @staticmethod
     def get_tasks(modules_only=False):
         """
         Get all the tasks available to the activator, basically it returns all the tasks defined as
         (Super)Task classes inside all the packages defined in TASK_PACKAGES
 
-        TODO: Use this function to locate package.module for a given (Super)Task  and use it
-        above to avoid code duplication
+        TODO: Use this function to locate package.module for a given (Super)Task  and use it above to avoid
+        code duplication in load_super_task
 
         :param modules_only: If True, return only the modules, not the tasks
         :return: a list of task or modules inside every package from TASK_PACKAGES
         """
 
         tasks_list = []
+        inheritance_list = []
         module_list = []
         for package in TASK_PACKAGES.itervalues():
             for _, module_name, _ in pkgutil.iter_modules(package.__path__):
-
                 task_module = package.__name__ + '.' + module_name
                 module_list.append(task_module)
                 if not modules_only:
                     classes_map = pyclbr.readmodule(module_name, path=package.__path__)
                     module_classes = [class_key for class_key in classes_map.keys() if
                                       classes_map[class_key].module == module_name]
-                    for mod_cls in module_classes:
-                        # Get all task except the base classes
-                        if mod_cls[-4:].upper() == 'TASK' and mod_cls not in [
-                            'WorkFlowParTask', 'WorkFlowSeqTask', 'Task', 'SuperTask',
-                            'WorkFlowTask', 'CmdLineTask']:
-                            tasks_list.append(task_module + '.' + mod_cls)
+                    for mod_cls_name in module_classes:
+                        # Get all tasks except the base classes, they need to be ended with 'Task'
+                        if mod_cls_name.upper().endswith("TASK") and mod_cls_name not in EXEMPTED_TASKS:
+                            # Add the task class to the list for the current module
+                            tasks_list.append(task_module + '.' + mod_cls_name)
+                            # This will indicate the inheritance of classes for the give task, this is used
+                            # only for reference when listing available tasks in the case some tasks are
+                            # defined as a CmdLineTasks, Tasks or SuperTasks. The -> will indicate the
+                            # inheritance
+                            inheritance_list.append("->".join(get_inheritance(classes_map[mod_cls_name])))
```

Comment:
yes, I cleaned this a little to avoid extra work if  `modules_ony` is True, this is for printing list of avaiable modules and tasks.